### PR TITLE
[Flow EVM] Add error message to the Tx executed event (bug fix)

### PR DESCRIPTION
**Pull request by ramtinms**

### CODE REVIEW

File: fvm/evm/types/events.go [hunk 1]
```diff
@@ -92,13 +92,13 @@ func (p *transactionEvent) ToCadence(location common.Location) (cadence.Event, e
...
 			cadence.NewField("errorCode", cadence.UInt16Type),
+			cadence.NewField("errorMessage", cadence.StringType),
 			cadence.NewField("gasConsumed", cadence.UInt64Type),
 			cadence.NewField("contractAddress", cadence.StringType),
 			cadence.NewField("logs", cadence.StringType),
 			cadence.NewField("blockHeight", cadence.UInt64Type),
 			// todo we can remove hash and just reference block by height (evm-gateway dependency)
 			cadence.NewField("blockHash", cadence.StringType),
-			cadence.NewField("errorMessage", cadence.StringType),
 			cadence.NewField("returnedData", cadence.StringType),
```
Comment (turbolent) [inline fvm/evm/types/events.go:102]: This also doesn't seem to be in the event declaration, same below

Comment (ramtinms) [reply fvm/evm/types/events.go:102]: good catch, updated at [341b455](https://github.com/onflow/flow-go/pull/6089/commits/341b455ede9355f5a8d7b6ef1015a702983e3317)

File: fvm/evm/stdlib/contract.cdc [hunk 2]
```diff
@@ -60,6 +62,13 @@ contract EVM {
         blockHeight: UInt64,
         // block hash in which transaction was included
         blockHash: String
+        /// captures the data that is returned from
+        /// the evm. For contract deployments
+        /// it returns the code deployed to
+        /// the address provided in the contractAddress field.
+        /// in case of revert, the smart contract custom error message
+        /// is also returned here (see EIP-140 for more details).
+        returnedData:
```
Comment (turbolent) [inline fvm/evm/stdlib/contract.cdc:71]: Missing the type of the field here

**Review by janezpodhostnik — APPROVED** [review]
This will require an EVM contract update on previewnet after the HCU!

**Review by sideninja — APPROVED** [review]

**Comment by turbolent** [comment]
State commitments need to be updated one last time

**Review by turbolent — APPROVED** [review]

**Pull request merged**